tests/test_report_api.py

```diff
@@ -0,0 +1,175 @@
+import heterocl as hcl
+import numpy as np
+import re
+import json
+import xmltodict
+import pathlib
+import imageio
+from .test_report_data.rptdisp import RptDisp
```

Comment:
Now I see what you mean. In this case, you should just use `hcl.rptdisp.RptDisp` instead. No need to include the class Python file.

Reply:
I get `ModuleNotFoundError: No module named 'hcl'` if I were to not do this.

Reply:
How did you use that? Since you already `import heterocl as hcl`, that error message shouldn't pop out.

Reply:
> How did you use that? Since you already `import heterocl as hcl`, that error message shouldn't pop out.

Oh sorry I did it as an import statement. Fixed now.